spec.html

```diff
@@ -27139,7 +27139,7 @@
     <!-- es6num="21.1.3" -->
     <emu-clause id="sec-properties-of-the-string-prototype-object">
       <h1>Properties of the String Prototype Object</h1>
-      <p>The String prototype object is the intrinsic object <dfn>%StringPrototype%</dfn>. The String prototype object is an ordinary object. The String prototype is itself a String object; it has a [[StringData]] internal slot with the value *""*.</p>
+      <p>The String prototype object is the intrinsic object <dfn>%StringPrototype%</dfn>. The String prototype object is an exotic String object and has the internal methods specified for such objects. It has a [[StringData]] internal slot with the value *""*.</p>
```

Comment:
[[GetOwnProperty]], [[OwnPropertyKeys]], and (soon) [[DefineOwnProperty]] are overridden for exotic string objects in https://tc39.github.io/ecma262/#sec-string-exotic-objects. This is needed to support `new String("test")[0] === "t"`, because the indexed properties aren't _real_ properties but instead are defined as _virtual_ properties, similar to how this is implemented in browsers.